katsdptelstate/test/test_telescope_state.py

```diff
@@ -504,6 +504,15 @@ def test_undecodable_bytes_in_key(self):
         self.assertEqual(self.ts.get(key_b), 'hello')
         self.assertEqual(self.ts.get(key_b[1:]), None)
 
+    def test_tab_completion(self):
+        self.ts.clear()
```

Comment:
Shouldn't be necessary. There are some lingering remnants in some tests, but these days the test setup should provide a blank telstate.

Reply:
Right.